airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftStagingS3Destination.java

```diff
@@ -79,7 +79,7 @@ public AirbyteConnectionStatus check(final JsonNode config) {
       final String outputSchema = super.getNamingResolver().getIdentifier(config.get("schema").asText());
       attemptSQLCreateAndDropTableOperations(outputSchema, database, nameTransformer, redshiftS3StagingSqlOperations);
       return new AirbyteConnectionStatus().withStatus(AirbyteConnectionStatus.Status.SUCCEEDED);
-    } catch (final ConnectionErrorException e) {
+    } catch (final ConfigErrorException e) {
       final String message = getErrorMessage(e.getStateCode(), e.getErrorCode(), e.getExceptionMessage(), e);
```

Comment:
This line wouldn't work right because `ConfigErrorException` doesn't have the methods `getStateCode`, `getErrorCode`, `getExceptionMessage`, right? 

It would want to look something like
```java
catch (final ConfigErrorException e) {
  AirbyteTraceMessageUtility.emitConfigErrorTrace(e, e.getDisplayMessage);
  return new AirbyteConnectionStatus()
    .withStatus(AirbyteConnectionStatus.Status.FAILED)
    .withMessage(e.getDisplayMessage);
}
```

Reply:
Thanks I've updated all of this code here and below